notebooks/official/explainable_ai/xai_text_classification_feature_attributions.ipynb

```diff
@@ -0,0 +1,844 @@
+{
```

Comment:
<p>I would drop separate references to Keras. Keras and TF are the same since 2.0</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1801/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>